proto/destination.proto

```diff
@@ -136,6 +136,8 @@ message DestinationProfile {
   // as retries.  Failured requests on retryable routes will not be retried if
   // there is no available budget.
   RetryBudget retry_budget = 2;
+  // The timeout to use if none is specified by the route.
+  google.protobuf.Duration default_timeout = 3;
```

Comment:
What are the situations where this would be set? Why wouldn't this just be represented as a Route?

Reply:
Convenience, in case your timeouts are the same for your service across most of your routes.  This is not essential and could be removed if you don't think it's useful.

Reply:
I think this sort of thing may be more appropriate in the user-facing configuration; but i'm skeptical here. Should there be a default response classifier? If we add labels should we have a default set of labels? etc...

API-wise, i'd be more inclined to say that the controller should explicitly encode a default route if it wishes to provide default policy (and similarly set the timeout value in routes that would not otherwise have it).